tasks/build.yml

```diff
@@ -20,6 +20,17 @@
     src="{{python_src_prefix}}/{{python_src_pkg}}"
     dest="{{python_src_prefix}}"
 
+- name: Patch code
```

Comment:
I'd probably change a few things here, to improve a bit on idempotency.

1. use the `get_url` module, and iterate along the patches urls. Register successful downloads
2. Use:
```
args:
  chdir: "{{python_src_prefix}}"
```
instead of `cd ...`
3. Using the `register` var from `get_url` apply the patche.
4. Not sure about removing the patches. IMO it is a good idea to actually know what has been applied, and also, that would break `get_url` idempotency.